apps/angular/src/app/features/home/home.component.html

```diff
@@ -0,0 +1 @@
+<camp-anime-table></camp-anime-table>
```

Comment:
We can use self-closing tags in such cases.
```suggestion
<camp-anime-table />
```